required/latex-lab/documentmetadata-support.dtx

```diff
@@ -246,88 +236,87 @@
       {
         \msg_warning:nnn{pdf}{unknown-standard}{#1}
       }
-    ,pdfmanagement .bool_gset:N = \g__pdfmanagement_active_bool
-    ,firstaidoff .clist_gset:N = \g__pdfmanagement_firstaidoff_clist
     ,testphase .multichoice:
     ,testphase / tagpdf .code:n =
       {
-        \tl_gput_right:Nn\g__pdfmanagement_testphase_tl
-          {
-            \RequirePackage{tagpdf}
-            \AddToDocumentProperties [document]{testphase/tagpdf}{loaded}
-          }
-      }
-    ,testphase / headings .code:n =
-      {
-        \tl_gput_right:Nn\g__pdfmanagement_testphase_tl
-          {
-            \RequirePackage{headings-testphase}
-            \AddToDocumentProperties [document]{testphase/headings}{loaded}
-          }
-      }
-    ,testphase / ptagging .code:n =
-      {
-        \tl_gput_right:Nn\g__pdfmanagement_testphase_tl
+        \tl_gput_right:Nn\g_@@_testphase_tl
           {
-            \AddToHook{class/after}
+            \file_if_exist_input:nF {tagpdf-latex-lab-testphase.ltx}
               {
-                \RequirePackage{ptagging-testphase}
-                \AddToDocumentProperties [document]{testphase/ptagging}{loaded}
+                \RequirePackage{tagpdf}
+                \AddToDocumentProperties [document]{testphase/tagpdf}{loaded}
+                \tagpdfsetup{activate,paratagging,interwordspace}
+                \AddToDocumentProperties [document]{tagging}{active}
+                \AddToDocumentProperties [document]{tagging/para}{active}
+                \AddToDocumentProperties [document]{tagging/interwordspace}{active}
```

Comment:
I wonder if that shouldn't be `testphase = phase-I`
and then `=phase-II` would load a file that enable headings list footnotes ... the moment they are ready